test/remote/gateways/remote_worldpay_test.rb

```diff
@@ -246,19 +293,19 @@ def test_transcript_scrubbing
   # def get_authorization
   #   assert_success(response = @gateway.purchase(@amount, @credit_card, @options))
   #   assert response.authorization
-  #   puts "auth: " + response.authorization
+  #   puts 'auth: ' + response.authorization
```

Comment:
Does this test pass now, out of curiosity? If not, based on the blame, maybe we should just ax it; seems like it's been dead for a long time, Jim.

Reply:
They do in theory, just takes time.